src/containers/my-cooperations/accept-cooperation-close/AcceptCooperationClosing.tsx

```diff
@@ -38,7 +91,32 @@ const AcceptCooperationClosing: React.FC<AcceptCooperationClosureProps> = ({
       }
       icon={<ErrorOutlineRounded />}
       title={t('titles.acceptCooperationClosing')}
-    />
+    >
+      {isInputShown ? (
```

Comment:
Let's extract it to separate variable, probably using `useMemo` hook.
You can have there `if` statemens and it will be easier to read and maintain.